testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/expiration/SessionExpirationTestCase.java

```diff
@@ -434,43 +438,57 @@ public void test(@ArquillianResource(SessionOperationServlet.class) @OperateOnDe
             } finally {
                 HttpClientUtils.closeQuietly(response);
             }
+        }
 
-            // Trigger timeout of sessionId
-            Thread.sleep(AbstractClusteringTestCase.GRACE_TIME_TO_REPLICATE);
+        // Trigger timeout of sessionId - accounts for session timeout (1s) and infinispan reaper thread interval (1s)
+        // so that test conditions are typically met within the first attempt
+        Thread.sleep(TimeUnit.SECONDS.toMillis(2));
```

Comment:
We probably want to use TimeoutUtil.adjust(...) here.

Reply:
@pferraro I intentionally did not – the following retry logic handles any machines slowness.

Reply:
That's true.